backend/src/main/java/com/example/cms/ticket/TicketController.java

```diff
@@ -69,25 +69,24 @@ public ResponseEntity<Set<Response>> getTicketResponses(
 
     @PutMapping("/{ticketId}")
     public ResponseEntity<?> updateTicketStatus(
-            @PathVariable UUID ticketId, @RequestBody TicketStatus ticketStatusToChangeTo) {
+            @PathVariable UUID ticketId,
+            @RequestBody TicketStatus ticketStatusToChangeTo,
+            @RequestParam Optional<UUID> token) {
```

Comment:
only contact request handlers can change status therefore there's no need to seek a valid token in this updateStatus mapping